src/app/screens/connectors/ConnectStart9/index.tsx

```diff
@@ -136,6 +136,7 @@ export default function ConnectStart9() {
       <div className="mt-6">
         <TextField
           id="lndconnect"
+          type="text"
```

Comment:
let's make this also a password field. 

lndconnect holds secrets and thus we can handle it as a password field I guess.